GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerTestCase/RepairTests.cs

```diff
@@ -23,7 +33,11 @@ public void FixesCorruptHeadSha()
 
             this.Enlistment.TryMountGVFS().ShouldEqual(false, "GVFS shouldn't mount when HEAD is corrupt");
 
-            this.Enlistment.Repair();
+            this.Enlistment.Repair(false);

```

Comment:
We usually use named parameters for constant values like
`this.Enlistment.Repair(confirm: false);`